app/src/main/java/com/framgia/music_49/data/source/SongDataSource.java

```diff
@@ -0,0 +1,9 @@
+package com.framgia.music_49.data.source;
+
+import com.framgia.music_49.data.source.remote.SongRemoteDataCallBack;
+
+public interface SongDataSource {
+    interface RemoteDataSource extends SongDataSource {
```

Comment:
comment lại giải thích tại sao lại extend SongDataSource e
```suggestion
public interface Remote extends SongDataSource {
```

Reply:
cái này để sau này khi e mở rộng interface local vào thì nó sẽ extend được các method của interface SongDataSource không cần phải khai báo lại nữa a